### DISCUSSION TITLE
Use Selectize for sites' "Language" setting

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#10810] [Ideas] [answered]
Just a small quality of life thing, but it'd be great if the "Language" dropdown in site settings pages could use Selectize (like Craft 4's new Money field's "Currency" setting).

**Comment by brandonkelly** [comment, marked as answer]
Good call; changed for the next release! (4f75b32834b3fe2a2efe1baf5a3d3e80ec5de860)

**Reply by mmikkel** [reply]
Awesome, thank you!

**Reply by brandonkelly** [reply]
Craft 4.0.0-beta.4 is out now with that change.
